packages/sdks/web-component/src/lib/descope-wc/BaseDescopeWc.ts

```diff
@@ -265,6 +270,7 @@ class BaseDescopeWc extends HTMLElement {
       'form',
       'client',
       'validate-on-blur',
+      'style-name',
```

Comment:
I think we need to support switching it in runtime
Currently this value can be set only on initial loading